evm/test/IntegrationStandalone.t.sol

```diff
@@ -184,8 +184,18 @@ contract TestEndToEndBase is Test, IRateLimiterEvents {
 
         vm.stopPrank();
 
+        // Get the TransferSent(bytes32) event to ensure it matches up with the TransferRedeemed(bytes32) event later
+        Vm.Log[] memory recordedLogs = vm.getRecordedLogs();
+        bytes32 sentEventDigest;
+        for (uint256 i = 0; i < recordedLogs.length; i++) {
+            if (recordedLogs[i].topics[0] == keccak256("TransferSent(bytes32)")) {
+                sentEventDigest = recordedLogs[i].topics[1];
```

Comment:
Should there be a `break` out of the loop once this assignment happens?

Reply:
fair. only in a test, so will cover in #522 if pushing again